lib/editor/util/timetable.js

```diff
@@ -40,12 +40,23 @@ export function getHeaderColumns (
   return columns.filter(c => c.type !== 'DEPARTURE_TIME')
 }
 
-export function parseTime (timeString: string) {
+/**
+* Handles pasted data from clipboard (e.g. from CSV file)
+* If departure/arrival time cell, pastes in time format, otherwise returns string as is
+ */
+let alertTimeout
+export function parseCellValue (timeString: string, col: TimetableColumn) {
   const date = moment().startOf('day').format('YYYY-MM-DD')
-  return moment(date + 'T' + timeString, TIMETABLE_FORMATS).diff(
-    date,
-    'seconds'
-  )
+  const parsedDate = moment(date + 'T' + timeString, TIMETABLE_FORMATS, true)
+  clearTimeout(alertTimeout)
+  if (isTimeFormat(col.type)) {
+    if (!parsedDate.isValid()) {
+      alertTimeout = setTimeout(() => alert('Please enter a valid time format'), 600)
```

Comment:
Why do we wait to alert the user?

Reply:
This prevented an issue where the alert would show twice, I can make the timeout shorter